From: martin f. krafft <madduck@debian.org>
Subject: Disable udev incremental assembly

Upstream enabled incremental assembly in the udev rules by default for 3.1.3,
but this is too early. Hence, this patch simply reverts the change.

Signed-off-by: martin f. krafft <madduck@debian.org>

---
 udev-md-raid.rules |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/udev-md-raid.rules b/udev-md-raid.rules
index f564f70..241c31d 100644
--- a/udev-md-raid.rules
+++ b/udev-md-raid.rules
@@ -8,6 +8,10 @@ GOTO="md_inc_skip"
 
 LABEL="md_inc"
 
+## DISABLED: Incremental udev assembly disabled
+## ** this is a Debian-specific change **
+GOTO="md_inc_skip"
+
 # remember you can limit what gets auto/incrementally assembled by
 # mdadm.conf(5)'s 'AUTO' and selectively whitelist using 'ARRAY'
 ACTION=="add", RUN+="/sbin/mdadm --incremental $tempnode"
